ga4gh/backends.py

```diff
@@ -280,30 +280,30 @@ def __init__(self, variantSetId, vcfPath):
         # self.tb chrom -> vcf dict
         self._tabixMap = {}
 
-        for fn in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
-            tabixfile = pysam.Tabixfile(fn)
+        for vcfFile in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
+            tabixfile = pysam.Tabixfile(vcfFile)
             for chrom in tabixfile.contigs:
                 if chrom in self._tabixMap:
                     raise Exception("cannot have overlapping VCF files.")
                 self._tabixMap[chrom] = tabixfile
 
     def convertVariant(self, record):
-        v = protocol.GAVariant()
```

Comment:
Going from v to variant adds a lot of verbosity to this function.  Given that it's called convertVariaint is v is really ambiguous?